README.md

```diff
@@ -15,7 +16,18 @@ Place your plugin features list.
 
 ## Usage
 
-Include a usage description for your plugin.
+To configure this plugin correctly you have to set this environment variables in your
+local machine like [here](https://github.com/Pearson-Advance/edx-deployment-pe/blob/juniper/ansible/environments/prod/group_vars/all/10_edxapp.yml#L47).
+
+## Testing
+
+- In Studio add a new video component to a course.
+- Press the *Edit* button and then go to the *Advanced* section.
+- In the *Video File URLs* field add the protected video. (e,g,. /secure-cloudfront-video/?key=9780136721390/M101_01_08_08_01.mp4).
```

Comment:
```suggestion
- In the *Video File URLs* field add the protected video. (e,g,. /secure-cloudfront-video/?key=9780136721390/M101_01_08_08_01.mp4).
```
This is a link to an actual protected resource, you **should never** link to a protected resource in a public repository. Just add an example of what the URL looks like. @sergivalero20